docs/pages/guides/apple-privacy.mdx

```diff
@@ -0,0 +1,62 @@
+---
+title: Configuring iOS Privacy Manifests
+description: Learn about configuring iOS privacy manifests for your mobile app.
+---
+
+If you're using one of React Native or Expo libraries that use one of the "restricted reason" APIs, you'll need to configure an iOS privacy manifest to declare why you're including native code calling those APIs.
+
+More details and a list of "required reason" APIs can be found in the [Apple Developer Documentation](https://developer.apple.com/documentation/bundleresources/privacy_manifest_files).
+
+> Note:
+> The information and steps included in this document are still in development and may change, either due to new tools built for this purpose or changes in Apple's requirements.
+
+## What is a Privacy Manifest?
+
+A privacy manifest is a file named `PrivacyInfo.xcprivacy` that is included in your iOS native project. This file is used to declare why you're including native code that calls certain APIs that Apple considers sensitive.
+
+Those APIs currently include accessing UserDefaults, file timestamp APIs, system boot time APIs, disk space APIs and active keyboard APIs, but Apple considers it an open list that can be expanded in the future.
+
+## Including an iOS Privacy Manifest in Expo Managed Apps
```

Comment:
```suggestion
## Configuration in app.json/app.config.js
```

Reply:
using a consistent title here with what we use in reference pages, eg: https://docs.expo.dev/versions/latest/sdk/camera/#configuration-in-appjsonappconfigjs